helm/values.yaml

```diff
@@ -0,0 +1,342 @@
+# Default values for mas.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: ghcr.io/matrix-org/matrix-authentication-service
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "0.9.0"
```

Comment:
why not use `appVersion` from the chart as default ([ref](https://gitlab.com/ananace/charts/-/blob/master/charts/matrix-synapse/Chart.yaml?ref_type=heads#L6))?